schematic/json_schema_functions.py

```diff
@@ -0,0 +1,368 @@
+"""JSON Schema functions
+This module includes functions for:
+- uploading JSON Schemas to Synapse
+- binding JSON Schemas to Synapse entities
+- creating file views and wikis based on JSON Schemas
+
+Several functions are deprecated. These are the ones that have to do with creating a fileview and
+ wiki in Synapse. This functionality is only needed temporarily for Curator purposes while the
+ Synapse platform creates this functionality internally. These will be sunsetted in the near future.
+
+"""
+
+from typing import Any, Optional
+from deprecated import deprecated
+
+from synapseclient import Synapse  # type: ignore
+from synapseclient.models import Column, ColumnType, ViewTypeMask, EntityView  # type: ignore
+from synapseclient import Wiki  # type: ignore
+
+from schematic.schemas.create_json_schema import create_json_schema
+from schematic.schemas.data_model_graph import create_dmge
+
+TYPE_DICT = {
+    "string": ColumnType.STRING,
+    "number": ColumnType.DOUBLE,
+    "integer": ColumnType.INTEGER,
+    "boolean": ColumnType.BOOLEAN,
+}
+
+LIST_TYPE_DICT = {
+    "string": ColumnType.STRING_LIST,
+    "integer": ColumnType.INTEGER_LIST,
+    "boolean": ColumnType.BOOLEAN_LIST,
+}
+
+
+@deprecated(reason="Entity view functionality is only need temporarily")
```

Comment:
Great addition of using `deprecated`, although we're just piggy backing off of the fact that synapseclient has this dependency.